public/content/eips/index.md

```diff
@@ -54,10 +54,19 @@ Full standardization process alongside with chart is described in [EIP-1](https:
 
 If you’re interested to read more about EIPs, check out the [EIPs website](https://eips.ethereum.org/) and [EIP-1](https://eips.ethereum.org/EIPS/eip-1). Here are some useful links:
 
+### Official EIP resources
```

Comment:
Ethereum Improvement Proposals are by and for the community. EIP editors and ECH team supports to make them publicly accessible at https://eips.ethereum.org/

My preference will be to refrain from "Official".

Reply:
> My preference will be to refrain from "Official".

I talked about this in [another comment](https://github.com/ethereum/ethereum-org-website/pull/12117#discussion_r1531415853), but I'll provide more context here. The rationale for "official" is to separate  what is the authoritative source of information about the specification, description, and status of EIPs/ERCs (eips.ethereum.org and ercs.ethereum.org) from other resources that make information about EIPs/ERCs easier to consume. For example, the [OMG Wiki](https://www.omgwiki.org/dido/doku.php?id=dido:public:ra:xapend:xapend.b_stds:defact:ethereum:eip:erc_0020) directly pulls information about EIPs from eips.ethereum.org, but has a notice like the following on every page: 

*"Note: The following is an excerpt from the official Ethereum site. It is provided here as a convenience and is not authoritative. Refer to the original document as the authoritative reference."*

AFAIK, the other resources I listed on the page would fall into the category of "providing information as a convenience" and wouldn't be authoritative. An example: it's possible the specification for an EIP changes just after the authors appear to discuss the EIP on PEEPanEIP; I write the EIPs For Nerds series and the specification for at least for one EIP has changed significantly since I started publishing deep dives on EIPs (which means I need to update to reflect what's displayed on the EIP's page on eips.ethereum.org). EIPs Insight and eips.wtf are probably exceptions, but it's sort of the difference between [erc4337.com](https://erc4337.com/) and [erc4337.io](https://www.erc4337.io/)--both are offering roughly the same information, but have different levels of authoritativeness.